src/main/java/com/synopsys/integration/alert/component/settings/SettingsDescriptor.java

```diff
@@ -76,6 +76,8 @@
     public static final String FIELD_ERROR_LDAP_SERVER_MISSING = "LDAP Server missing";
     public static final String FIELD_ERROR_LDAP_USERNAME_MISSING = "LDAP Manager DN is missing";
     public static final String FIELD_ERROR_LDAP_PASSWORD_MISSING = "LDAP Manager DN password is missing";
+    public static final String FIELD_ERROR_LDAP_AUTHENTICATION_TYPE_INVALID = "Invalid LDAP Authentication type specified.";
+    public static final String FIELD_ERROR_LDAP_REFERRAL_INVALID = "Invalid LDAP Referral specified";
```

Comment:
If we move the validation to the SelectConfigField then we should be able to generate these errors from the data we already have.